src/ucp/core/ucp_worker.c

```diff
@@ -1699,10 +1700,21 @@ static void ucp_worker_print_used_tls(const ucp_ep_config_key_t *key,
     p += strlen(p);
 
     for (lane = 0; lane < key->num_lanes; ++lane) {
-        if (((key->am_lane == lane) || (lane == key->tag_lane) ||
+        if ((key->am_lane == lane) ||
             (ucp_ep_config_get_multi_lane_prio(key->am_bw_lanes, lane) >= 0)  ||
-            (ucp_ep_config_get_multi_lane_prio(key->rma_bw_lanes, lane) >= 0)) &&
-            (context->config.features & UCP_FEATURE_TAG)) {
+            (ucp_ep_config_get_multi_lane_prio(key->rma_bw_lanes, lane) >= 0)) {
+            if (context->config.features & UCP_FEATURE_TAG) {
+                tag_lanes_map |= UCS_BIT(lane);
+            }
+
+            if (context->config.features & UCP_FEATURE_AM) {
+                am_lanes_map |= UCS_BIT(lane);
+            }
+        }
```

Comment:
seems we need only one mask of lanes, and lines 1735..1742 which should check feature flags

Reply:
but different features may use different lanes

Reply:
but the masks will be the same anyway value no (unless they are 0 because feature is off)?

Reply:
ok, you mean print `am_bw_lanes`, `rma_bw_lanes`, etc and used features separately? 
If yes, it may be hard for users to understand what lanes are used for what feature